lib/hls/hls_parser.js

```diff
@@ -3181,7 +3186,8 @@ shaka.hls.HlsParser = class {
    * @param {string} mimeType
```

Comment:
The description for this method should now mention that it is also computing the average bitrate from EXT-X-BITRATE.

Reply:
Changed!